docs/shared/recipes/root-level-scripts.md

```diff
@@ -0,0 +1,71 @@
+# Run Root-Level NPM Scripts with Nx
+
+There are often tasks in a codebase that apply to the whole codebase rather than a single project. Before version 15.3.0, the only way to use Nx's task pipelines or caching mechanisms for those scripts was to create an empty `scripts` project and run the scripts from there. Now you can run those commands directly from the root `package.json`.
+
+## Example
+
+Let's say your root `package.json` looks like this:
+
+```json
+{
+  "name": "myorg",
+  "scripts": {
+    "echo": "echo hello world"
+  }
+}
+```
+
+We want to be able to run the `echo` script using Nx.
+
+## Setup
+
+There are two methods to make Nx aware of the root `package.json` scripts. You can either:
+
+1. Add an `"nx": {}` property to the root `package.json` -OR-
+2. Create a small `project.json` file at the root with the same name as defined in your root `package.json`.
```

Comment:
Id' not mention the project.json bit cause I think we would always recommend against it.